mica/archive/aca_l0.py

```diff
@@ -465,7 +466,7 @@ def _get_missing_archive_files(self, start, only_new=False):
                     startdate)
         # find the index in the cda archive list that matches
         # the first entry with the "start" date
-        for idate, backcnt in zip(ingested_files[::-1]['ingest_date'],
+        for idate, backcnt in zip(ingested_files['ingest_date'][::-1],
```

Comment:
Looks like I just wasn't paying attention to order when I put this "back" after doing more of the "unicode sandwich" but this order is probably better anyway.